firebase.json

```diff
@@ -20,11 +20,11 @@
     ],
     "headers": [
       {
-        "source": "**/*.@(css|html|js|json|txt|xml|markdown|pdf)",
+        "source": "**/*.@(css|js|json|txt|xml|markdown|pdf)",
         "headers": [
           {
             "key": "Cache-Control",
-            "value": "max-age=315360000"
+            "value": "max-age=0, no-cache"
```

Comment:
@ozasadnyy it works fine with this configuration. But if you want we can try to get rid of it :)